src/display/text_layer.js

```diff
@@ -197,6 +223,35 @@ function appendText(task, geom, styles, ctx) {
   }
 }
 
+function layout(params) {
+  const { div, scale, properties, ctx, prevFontSize, prevFontFamily } = params;
```

Comment:
Nit: These lines could be combined, with the destructuring happening directly above.

Reply:
I did that but I need to save the `prev...`:
https://github.com/mozilla/pdf.js/pull/15722/files#diff-eb5220436f55f0c6ff687c15781981b323be818cb77e49d53d66f7f372f1f23dR236-R237